src/main/java/org/javarosa/form/api/FormEntryController.java

```diff
@@ -29,6 +29,7 @@
     public static final int EVENT_REPEAT_JUNCTURE = 32;
 
     public final static String FIELD_LIST = "field-list";
```

Comment:
Can you add a comment above these 2 fields indicating that they are both appearance attributes for a group?